### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by ruebeckscube** [ruebeckscube/localmusic#99]
Everything should be accessible by default since we're mostly using semantic inputs and good labels, probably the only thing that requires direct aria-labeling is the date picker widget (and maybe the menu that happens when you log in).

**Comment by ruebeckscube** [comment]
a84af97
